components/ntp_background_images/browser/view_counter_service.h

```diff
@@ -139,6 +139,9 @@ class ViewCounterService : public KeyedService,
   // Should we show the branded wallpaper right now, in addition
   // to the result from `IsBrandedWallpaperActive()`.
   bool ShouldShowBrandedWallpaper() const;
+  // Should we show custom background that user uploaded or fill background with
```

Comment:
I think this comment is suitable for `NTPCustomBackgroundImagesService::ShouldShowCustomBackground()`?
and `ViewCounterService::ShouldShowCustomBackgroud()` should be `ViewCounterService::ShouldShowCustomBackgroundImages()`?

IMO, changes in `ViewCountrService` seems not give much benifit. WDYT?

Reply:
Oops, I was confusing. I thought this doesn't have logic change.
Got it. `NTPCustomBackgroundImagesService::ShouldShowCustomBackground()` has changed. Thanks.